cmd/test/validationManager.go

```diff
@@ -63,6 +85,14 @@ func (v *ValidationManager) ValidK8sConfigurationsCount() int {
 	return totalConfigs
 }
 
+func (v *ValidationManager) SaveK8sValidationWarningPerValidFile(k8sValidationWarningPerValidFile *validation.K8sValidationWarningPerValidFile) {
```

Comment:
Save has an implicit meaning of persistence, it is more appropriate to call this `SetK8sValidationWarningPerValidFile`